test/test_xbuilder.cpp

```diff
@@ -286,6 +286,15 @@ namespace xt
         ASSERT_TRUE(all(equal(ls, expected)));
     }
 
+    TEST(xbuilder, linspace_double)
+    {
```

Comment:
No idea why this test is at the end of the file. Just move it to stay together with other `linspace` test.

Reply:
No idea either, the tests were probably added in different PRs and we didn't pay attention to organize them properly. Thanks for this.